lib/dockerfile.ts

```diff
@@ -282,27 +199,35 @@ class Dockerfile {
 			return command;
 		}
 
-		if (_.isObject(command)) {
-			throw new DockerfileParseError(
-				'Object arguments not supported for RUN commands',
-			);
+		if (_.isArray(command)) {
+			return (command as string[]).join(' ');
```

Comment:
This cast shouldn't be necessary as the `isArray` should already narrow this to being just `string[]`

Reply:
Bump